app/controllers/grade_entry_forms_controller.rb

```diff
@@ -382,13 +383,14 @@ def csv_upload
         if totals.empty?
           totals = row
           # Create/update the grade entry items
-          GradeEntryItem.create_or_update_from_csv_rows(
+          grades = GradeEntryItem.create_or_update_from_csv_rows(
             names,
             totals,
             @grade_entry_form,
             overwrite)
           next
         end
+        GradeEntryItem.import grades
```

Comment:
@rathukugan this line should be moved to inside the previous block (before the `next`).